### DISCUSSION TITLE
Project site while using CNAME on Org site

### DISCUSSION THREAD

**Discussion by astrojuanlu** [community/community#22933] [Other Feature Feedback, Questions, & Ideas]
<p>I have an Org site at <a href="https://blog.poliastro.space" rel="nofollow noopener">https://blog.poliastro.space</a> with this repository:</p>
<p><a href="https://github.com/poliastro/poliastro.github.io/tree/master" rel="nofollow noopener">https://github.com/poliastro/poliastro.github.io/tree/master</a></p>
<p>And a CNAME in my poliastro.space domain like this:</p>
<p><a href="https://mxtoolbox.com/SuperTool.aspx?action=a%3apoliastro.space&amp;run=toolpage" rel="nofollow noopener">https://mxtoolbox.com/SuperTool.aspx?action=a%3apoliastro.space&amp;run=toolpage</a></p>
<p>Now I’d like to setup this project site:</p>
<p><a href="https://github.com/poliastro/poliastro-benchmarks/tree/gh-pages" rel="nofollow noopener">https://github.com/poliastro/poliastro-benchmarks/tree/gh-pages</a></p>
<p>But if I try to browse <a href="https://poliastro.github.io/poliastro-benchmarks" rel="nofollow noopener">https://poliastro.github.io/poliastro-benchmarks</a> it redirects to <a href="https://blog.poliastro.space/poliastro-benchmarks" rel="nofollow noopener">https://blog.poliastro.space/poliastro-benchmarks</a>, which yields a 404.</p>
<p>This is very similar to the issue reported here:</p>
<p><a href="https://github.com/isaacs/github/issues/547#issuecomment-220288687" rel="nofollow noopener">https://github.com/isaacs/github/issues/547#issuecomment-220288687</a></p>
<p>But the responses there do not clarify my problem. In particular, according to a GitHub support person:</p>
<p>&gt; Yep the <a href="http://user.exmaple.com/projectname" class="inline-onebox-loading" rel="nofollow noopener">user.exmaple.com/projectname</a> will automagically serve user/projectname gh-pages content.</p>
<p>But apparently it’s not working in my case. Anything I’m doing wrong?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi there <a class="mention" href="/juanlu001">@juanlu001</a>,</p>
<p>Thanks for being part of the Community Forum. I’ll help as best I can.</p>
<p>If you’ve never published to GitHub Pages before from a particular repo, then things should magically start publishing when you create a <code>gh-pages</code> branch. But if you previously selected some other branch that doesn’t exist anymore, you may have to go to the repo settings and select <code>gh-pages</code> here:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/0f7b3d1ddb313732e11dd3ad901bda662f556622.png" data-download-href="/uploads/short-url/2cXco9Ih8cpKO2ROLbolr4WhFYu.png?dl=1" title="Screen_Shot_2018-06-07_at_11_48_52_AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/16922726/181073746-374ecd24-210e-422c-890d-0e6f9a183db6.png" alt="Screen_Shot_2018-06-07_at_11_48_52_AM.png" data-base62-sha1="2cXco9Ih8cpKO2ROLbolr4WhFYu" width="646" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/0f7b3d1ddb313732e11dd3ad901bda662f556622_2_646x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/0f7b3d1ddb313732e11dd3ad901bda662f556622_2_969x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/0/0f7b3d1ddb313732e11dd3ad901bda662f556622_2_1292x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/0f7b3d1ddb313732e11dd3ad901bda662f556622_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen_Shot_2018-06-07_at_11_48_52_AM.png</span><span class="informations">1884×1458 274 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If that still doesn’t do the trick, I’d recommend reaching out to private support at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> so we can dig a little deeper.</p>
<p>Cheers!</p>

**Comment by astrojuanlu** [comment]
<p>Hi Nadia,</p>
<p>Thanks for your help! The repository was indeed properly configured, but to make sure I made a commit to gh-pages and… The pages appeared! Problem solved.</p>
<p>Best,</p>
